### PR TITLE
Widgets RELEASE

### PR DESCRIPTION
**Pull request by nirgur**
## Related Issues

Fixes <link_to_github_issue>

## Related PRs

| branch       | PR         |
| ------------ | ---------- |
| service a PR | Link to PR |
| service b PR | Link to PR |

## Description

A few sentences describing the overall goals of the pull request's commits.

## Must

- [ ] Tests
- [ ] Documentation (if applicable)

### CODE REVIEW

File: example/components/ManageAccessKeys.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!-- eslint-disable vue/multi-word-component-names -->
+<template>
+	<div class="manage-roles-wrapper">
```
Comment (dorsha) [inline example/components/ManageAccessKeys.vue:3]: should be access keys no?

File: example/components/ManageAccessKeys.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</template>
+
+<script setup>
+import { AccessKeyManagement } from '../../src';
+
+const tenant = process.env.VUE_APP_DESCOPE_TENANT;
+</script>
+
+<style>
+.manage-roles-wrapper {
```
Comment (dorsha) [inline example/components/ManageAccessKeys.vue:16]: access keys

File: tests/AccessKeyManagement.test.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+jest.mock('../src/hooks', () => ({
+	useOptions: () => ({ projectId: 'project1', baseUrl: 'baseUrl' }),
+	useDescope: () => ({ httpClient: { hooks: { afterRequest: jest.fn() } } }),
+	useUser: () => ({}),
+	useSession: () => ({})
+}));
+
+globalThis.Response = <any>class {};
+
```
Comment (dorsha) [inline tests/AccessKeyManagement.test.ts:12]: Lint issue

**Review by dorsha — COMMENTED** [review]
Cool!

File: example/components/ManageAccessKeys.vue (hunk 1 above)
Comment (nirgur) [reply example/components/ManageAccessKeys.vue:3]: fixed

File: example/components/ManageAccessKeys.vue (hunk 2 above)
Comment (nirgur) [reply example/components/ManageAccessKeys.vue:16]: fixed

File: tests/AccessKeyManagement.test.ts (hunk 3 above)
Comment (nirgur) [reply tests/AccessKeyManagement.test.ts:12]: it's in the test
added ignore

**Review by dorsha — APPROVED** [review]

**Pull request merged**